types/geojson/index.d.ts

```diff
@@ -62,6 +63,11 @@ export interface GeoJsonObject {
     bbox?: BBox;
 }
 
+/**
+ * Union of GeoJSON objects.
+ */
+export type Object = Geometry | Feature | FeatureCollection;
```

Comment:
All GeoJSON Objects are extended from `GeoJsonObject`, so `Object` is just the same thing vice-versa, isn't? Why do you want to add this type? In addition, the naming `Object` for this type is a bad choice, because there is already the system-wide `Object` type.

Reply:
I still not happy with the name `Object`. What is your opinion on that @Cobster and @JeffJacobson? Of course we cannot use `GeoJSONObject`, because it is already the name of the interface. Maybe something like `AnyGeoJSONObject` or `GeoJSONObjectType`, but this is quite a long name.